src/template.rs

```diff
@@ -89,10 +88,10 @@ impl Template {
         self
     }
 
-    pub fn set_year(mut self, year: &str) -> Template {
-        self.context.year = Some(year.to_string());
-        self
-    }
+    // pub fn set_year(mut self, year: &str) -> Template {
```

Comment:
If it's unused just delete it. That is preferrable to having commented out code.

Reply:
Fixed in 
https://github.com/chasinglogic/licensure/pull/13/commits/5256f585883da0009bbd2a873153272ccb4425e5